builds.sh

```diff
@@ -160,11 +184,20 @@ for sourceId; do
 			buildIdJson="$(jq <<<"$buildIdParts" -c 'del(.resolvedParents)')" # see notes above (where buildIdParts is first defined)
 			buildId="$(sha256sum <<<"$buildIdJson" | cut -d' ' -f1)" # see notes above (where buildIdParts is first defined)
 			printf >&2 '%s\n' "$buildId"
+
 			img="$BASHBREW_STAGING_TEMPLATE"
 			img="${img//BUILD/$buildId}"
 			[ "$img" != "$BASHBREW_STAGING_TEMPLATE" ] # BUILD is required, for proper uniqueness (ARCH is optional)
 			img="${img//ARCH/$arch}"
-			resolved="$(_resolve "$img" "$arch")"
+
+			if [ -v "imageArchResolved["$img"]" ]; then
+				remoteArches="${imageArchResolved["$img"]}"
```

Comment:
```suggestion
				remoteArches=${imageArchResolved[$img]}
```

Reply:
Same comment as above, I prefer this one as is in the PR.